Marlin/ultralcd.cpp

```diff
@@ -2315,9 +2339,9 @@ void kill_screen(const char* lcd_msg) {
     void _reset_e_acceleration_rate(const uint8_t e) { if (e == active_extruder) _reset_acceleration_rates(); }
     void _reset_e0_acceleration_rate() { _reset_e_acceleration_rate(0); }
     void _reset_e1_acceleration_rate() { _reset_e_acceleration_rate(1); }
-    #if E_STEPPERS > 2
+    #if EXTRUDERS > 2
```

Comment:
there is no defenition of E_MANUAL in conditionals_LCD.h ... What is the meaning of the parameter ?